apps/server/src/shared/domain/entity/boardnode/boardnode.entity.ts

```diff
@@ -1,10 +1,9 @@
 import { Entity, Enum, Index, Property } from '@mikro-orm/core';
 import { InternalServerErrorException } from '@nestjs/common';
-import { AnyBoardDo } from '../../domainobject';
-import { EntityId } from '../../types';
+import { AnyBoardDo } from '@shared/domain/domainobject';
```

Comment:
Why do you this changes? It is not partial of your PR?

Reply:
I had a circular import dependency, which is why I re-did the imports in some related files. "Dependency cycle via ../entity:1=>./all-entities:2=>./boardnode/boardnode.entity:6(import/no-cycle)"